documentation/vcp-roles.md

```diff
@@ -0,0 +1,112 @@
+---
+title: Customize roles and privileges for different user cases
+---
+
+## Minimal set of vSphere roles/privileges required for static only persistent volume provisioning.
+
+**Note Datastore.FileManagement is only required for the role `manage-k8s-volumes`, if PVC is created to bind with statically provisioned PV, and reclaim policy set to delete. When PVC is deleted, associated statically provisioned PV will also be deleted.**
+
+<table>
+<thead>
+<tr>
+  <th>Roles</th>
+  <th>Privileges</th>
+  <th>Entities</th>
+  <th>Propagate to Children</th>
+</tr>
+</thead>
+<tbody><tr>
+  <td>manage-k8s-node-vms</td>
+  <td>System.Anonymous<br> System.Read<br> System.View<br> VirtualMachine.Config.AddExistingDisk<br> VirtualMachine.Config.AddNewDisk<br> VirtualMachine.Config.AddRemoveDevice<br> VirtualMachine.Config.RemoveDisk<br>
+  <td>VM Folder</td>
+  <td>Yes</td>
+</tr>
+<tr>
+  <td>manage-k8s-volumes</td>
+  <td>Datastore.FileManagement (Low level file operations
+)<br> System.Anonymous<br> System.Read<br> System.View<br>
```

Comment:
lets remove `System.Anonymous` `System.Read` and  `System.View` from all roles. This comes by default when you add any other privileges.  Also this is not visible on the User Interface, thus confusing end user.

Reply:
Sure, will fix.